pkg/utils/utils.go

```diff
@@ -474,6 +474,15 @@ func GetPfEswitchMode(pciAddr string) (string, error) {
 	return devLinkDeviceAttrs.Mode, nil
 }
 
+// HasRdmaParam returns true if PCI device has "enable_rdma" param
+func HasRdmaParam(pciAddr string) (bool, error) {
```

Comment:
this is called by rdmaSpec with deviceID which is either pci address or Auxiliary device name.

we should support both or add a new func that handles auxiliary devices

Reply:
Updated to support both